frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -337,6 +346,10 @@ public static function apiResolve(\OmegaUp\Request $r) {
         return ['status' => 'ok'];
     }
 
+    /**
+     * @param string $problemUrl
+     * @return null|string
+     */
     public static function extractAliasFromArgument($problemUrl) {
```

Comment:
siempre que puedas agregar los tipos en la firma de la función, hay que hacerlo:

```suggestion
     public static function extractAliasFromArgument(string $problemUrl) : ?string {
```

así no necesitas poner el docstring! (en algunos casos)